src/phpDebug.ts

```diff
@@ -285,7 +285,7 @@ class PhpDebugSession extends vscode.DebugSession {
                         this.sendErrorResponse(response, error)
                         reject(error)
                     })
-                    this._proxyConnect.sendProxyInitCommand()
+                    this._proxyConnect.sendProxyInitCommand(args.port || 9000)
```

Comment:
Why not pass this to the `ProxyConnect` like the other config? That would mean you'd have access in the constructor for the message

Reply:
Personal preference, I guess, did change it however.